docs/event-bus/docs/050-subscription-updates.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Subscription updates
+type: Details
+---
+
+The subscription crd can be updated using kubectl:
```

Comment:
`To update the subscription CRD, run this command: ` 

I would use the similar wording that we have in the CRD description to keep the consistency.